lib/pages/admin_pipeline_page.rb

```diff
@@ -38,6 +42,66 @@ def move_pipeline(pipeline, _source_group, destination_group)
       page.find('#div_move_to_groups').find('ul li.move_to_group_option', {text: destination_group}).click
     end
 
+    def verify_number_of_error_message(number_of_err_msg)
+        wait_till_event_occurs_or_bomb 60, "Total number of errors are not equal to #{number_of_err_msg}" do
+        reload_page
+        break if number_of_err_msg.to_i == error_and_warning_count.text.split("and")[0].scan(/\d+/)[0].to_i
+          end
+    end  
+
+    def verify_number_of_warnings(number_of_warning_msg)
+      if error_and_warning_count.text.split('and')[1].include? "warning"
+        total_warnings = error_and_warning_count.text.split('and')[1].scan(/\d+/)[0].to_i
+        assert_equal total_warnings, number_of_warning_msg.to_i
+       else 
+       assert_true false 
+      end 
+    end
+
+    def verify_there_are_no_warnings
+      assert_true !error_and_warning_count.text.include?("Warning")
+    end  
+    
+    def verify_there_are_no_error_messages
+      assert_true !error_and_warning_count.text.include?("error")
```

Comment:
Shouldn't the text include "Error"? since the text associated with the errors and warnings element is  with caps for first letter. This assertion will pass even when there are errors